build.gradle

```diff
@@ -128,7 +128,7 @@ jacocoTestReport {
     }
 }
 
-String version = '7.3.1'
+String version = '7.3.2'
```

Comment:
Hey @peterzhuamazon you can remove this as I have change pushed which has `7.3.2`.
https://github.com/opensearch-project/opensearch-build-libraries/pull/559
Thanks

Reply:
Thanks 👍